systest/multi-tenancy/upgrade_basic_test.go

```diff
@@ -0,0 +1,743 @@
+//go:build integration || upgrade
```

Comment:
I missed to change this to //go:build upgrade.

I have changed this in my local copy, will send it for review with the subsequent draft.